scale-decode/src/visitor/decode.rs

```diff
@@ -123,10 +123,17 @@ impl<'temp, 'scale, 'resolver, V: Visitor> ResolvedTypeVisitor<'resolver>
         let res = self.visitor.visit_composite(&mut items, self.type_id);
 
         // Skip over any bytes that the visitor chose not to decode:
-        items.skip_decoding()?;
-        *self.data = items.bytes_from_undecoded();
+        let skip_res = items.skip_decoding();
```

Comment:
Is it possible to extract "this logic" to a helper function that we can re-use instead repeating it for the other visitors?

Reply:
the types are all a bit different and just happen to have the same method names; it could be a macro though :)